### PR TITLE
Improve failed job ui

### PR DESCRIPTION
**Pull request by aaronlifton2**
At our company, we heavily rely on qless and sometimes we have a lot of failed jobs, resulting in a broken UI.

So, I made the failed list in the subnav toggleable and scrollable.
Also I renamed `style.css` to `resque.css`, since it's not currently being used, and moved css that isn't part of bootstrap out of `docs.css` and into a new `style.css`

**Before**
screen shot 2017-12-03 at 11 22 38 pm

**After**

qless1

qless2

Also made a dev helper to create seed data that might make it easier for others to help develop the web interface.

### CODE REVIEW

File: lib/qless/server/static/css/docs.css [hunk 1]
```diff
@@ -815,25 +815,4 @@ form.well {
...
-.failed-job > .min-col-size {
-  min-width: 395px;
-}
-
-.failed-job .row {
-  margin-left:0;
-  margin-right:10px;
-  height: 30px;
-}
+}
```
Comment (b4hand) [inline lib/qless/server/static/css/docs.css]: Can you add back the trailing newline character on this line?

File: utils/dev/qless-web-dev [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+require 'qless/server'
+
+client = Qless::Client.new
+
+# Create mock data
+QlessWebDevHelper.create_mock_jobs(client)
+
+Vegas::Runner.new(Qless::Server.new(client), 'qless-web', {
+  foreground: true
+})
```
Comment (b4hand) [inline utils/dev/qless-web-dev]: Here's another missing trailing newline.

File: utils/dev/qless_web_dev_helper.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        job.perform if job
+      }
+    end
+
+    private
+    def random_jid
+      ('a'..'z').to_a.shuffle[0,8].join
+    end
+  end
+end
```
Comment (b4hand) [inline utils/dev/qless_web_dev_helper.rb]: Here's another missing trailing newline.

File: lib/qless/server/static/css/docs.css (hunk 1 above)
Comment (aaronlifton2) [reply lib/qless/server/static/css/docs.css]: sure

File: utils/dev/qless-web-dev (hunk 2 above)
Comment (aaronlifton2) [reply utils/dev/qless-web-dev]: thanks

File: utils/dev/qless_web_dev_helper.rb (hunk 3 above)
Comment (aaronlifton2) [reply utils/dev/qless_web_dev_helper.rb]: thanks

**Comment by b4hand** [comment]
If you make those changes and push up an update, I'll merge this branch. If you don't do it by tomorrow, I may just go ahead and merge and then fix the newlines myself.

**Comment by aaronlifton2** [comment]
sounds good, i'll fix them tonight. thanks!

**Pull request merged**